src/clarity-angular/checkboxes/checkbox.ts

```diff
@@ -46,6 +47,10 @@ export class Checkbox implements ControlValueAccessor {
     @Input("name")
     public name: string = null;
 
+    // If the host is disabled we apply it to the checkbox
+    @Input("clrDisabled")
+    public disabled: boolean = false;
```

Comment:
I think along with setting the `:disabled` property, we also set a class `.disabled` on the checkbox because that affects the label styles for the text too. I don't remember if we need to do both but we should double check.